src/test/java/io/polivakha/mojo/properties/WriteProjectPropertiesMojoTest.java

```diff
@@ -0,0 +1,137 @@
+package io.polivakha.mojo.properties;
+
+import org.apache.maven.model.Profile;
+import org.apache.maven.project.MavenProject;
+import org.junit.Before;
+import org.junit.Test;
+
+import java.io.BufferedReader;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.List;
+import java.util.Properties;
+
+import static org.junit.Assert.assertEquals;
+
+public class WriteProjectPropertiesMojoTest {
+
+    private MavenProject mavenProject;
+    private WriteProjectProperties writeProjectProperties;
+    private WriteActiveProfileProperties writeActiveProfileProperties;
+
+    @Before
+    public void setUp() {
+        mavenProject = new MavenProject();
+        writeProjectProperties = new WriteProjectProperties();
+        writeProjectProperties.setProject(mavenProject);
+        writeActiveProfileProperties = new WriteActiveProfileProperties();
+        writeActiveProfileProperties.setProject(mavenProject);
+    }
+
+
+    @Test
+    public void givenSystemProperties_whenWritePropertiesWithSortASC_thenPropertiesInFileASC() throws Exception {
+        Path temp = Files.createTempDirectory("mytmp");
+        Path file = Files.createFile(Paths.get(temp.toString(), "properties-from-pom.properties"));
+        writeProjectProperties.setOutputFile(file.toFile());
+        writeProjectProperties.setComment("Properties");
+        writeProjectProperties.setSort(true);
+        mavenProject.getProperties().setProperty("some.version", "1234");
+        mavenProject.getProperties().setProperty("another.version", "9098");
```

Comment:
Just to make sure, can you please add one more property here, to be sure that sorting was applied

Reply:
Hm, okay, no problem